### PR TITLE
Fixes #482, changes design of search settings

### LINKED ISSUE #482: Design changes in search settings
**Actual Behaviour**

There are some errors with the design of the search page, it differs from the Google search page in 

- margins and spacing

- The width of the content

- font

- title

image

**Expected Behaviour**
This is how the page should look:
image

### PR DESCRIPTION
**Pull request by Marauderer97**
Fixes issue #482 

Changes: Design of search settings page to match the market leader in the following ways:

- [x] Adds a title

- [x] changes spacing and margins

- [x] changes font(size and family)

- [x] changes text width

Demo Link: [Here](https://marauderer97.github.io/susper.com/)

Screenshots for the change: 

Before:
image
After:
image

### CODE REVIEW

**Comment by harshit98** [comment]
@Marauderer97 

- Do proper alignment. Compare it with market leader.
screenshot from 2017-06-15 19-46-17

- Align the checkbox properly.
screenshot from 2017-06-15 19-46-42

- Design the checkbox buttons similar to Google and align it properly.
screenshot from 2017-06-15 19-47-05

- Create a `save` and `cancel` button.
screenshot from 2017-06-15 19-47-53

File: src/app/searchsettings/searchsettings.component.html [hunk 1]
```diff
@@ -32,7 +32,8 @@
           <li><a href="#">Help</a></li>
         </ul>
     </div>
-    <div class="col-md-9">
+    <div class="rightPane col-md-9">
+      <div class="rightContent">
         <div>
           <h4><strong>SafeSearch Filters</strong></h4>
           <p>SafeSearch can help you block inappropriate or explicit images from your Google Search results. The SafeSearch filter isn’t 100% accurate, but it helps you avoid most violent and adult content.</p>
```
Comment (harshit98) [inline src/app/searchsettings/searchsettings.component.html]: Google Search results = > Susper Search results

**Comment by Marauderer97** [comment]
@harshit98 Please check now. Changes I have made

- Changed the word "Google" to "Susper" everywhere

- Aligned the heading properly

- Added Save and Cancel buttons

- Designed the radio buttons following Google. In addition, have also implemented the logic that just one radio button is selected at once

- Aligned the checkbox properly

image

**Comment by Marauderer97** [comment]
@harshit98 Codacy says gradients are missing for old webkit, but I have taken the css gradients from google directly, Google has not provided any gradients for the old kit, only for the new one, so I followed the same pattern

**Pull request merged**